Twig/PrivacyCookieTwigExtension.php

```diff
@@ -18,29 +20,25 @@ class PrivacyCookieTwigExtension extends Twig_Extension
      * we must inject service_container this way
      * @link https://github.com/symfony/symfony/issues/2347
      *
-     * @var $container \Symfony\Component\DependencyInjection\ContainerInterface
+     * @var \Symfony\Component\DependencyInjection\ContainerInterface
      */
     protected $container;
```

Comment:
To avoid passing container to extension you can use `needs_environment` option as an argument to `Twig_Function_Method`.